src/installer/Directory.Build.targets

```diff
@@ -149,4 +149,28 @@
       <LatestCommit>N/A</LatestCommit>
     </PropertyGroup>
   </Target>
+
+ <!--
+    SDK currently does not know about net6.0 TargetFramework.
+    We need to manually add these to correctly resolve the targeting packs.
+    https://github.com/dotnet/sdk/issues/13950  
+  -->  
+  <ItemGroup>
+    <KnownFrameworkReference Include="Microsoft.NETCore.App"
+      DefaultRuntimeFrameworkVersion="5.0.0"
+      IsTrimmable="true"
+      LatestRuntimeFrameworkVersion="5.0.0"
+      RuntimeFrameworkName="Microsoft.NETCore.App"
+      RuntimePackNamePatterns="Microsoft.NETCore.App.Runtime.**RID**"
+      RuntimePackRuntimeIdentifiers="linux-arm;linux-arm64;linux-musl-arm64;linux-musl-x64;linux-x64;osx-x64;rhel.6-x64;tizen.4.0.0-armel;tizen.5.0.0-armel;win-arm;win-arm64;win-x64;win-x86;ios-arm64;ios-arm;ios-x64;ios-x86;tvos-arm64;tvos-x64;android-arm64;android-arm;android-x64;android-x86;browser-wasm"
+      TargetFramework="$(NetCoreAppCurrent)"
```

Comment:
Doesn't this whole block suggest that we would use a a 5.0.0 targeting pack when compiling against net6.0 in installer? This doesn't make any sense to me but I hope that I'm misunderstanding something here.

Reply:
I suspect this is due to the installer tests saying they "target 6.0" but aren't set up correctly to use the live bits nor run on the them either?  @Anipik mentioned something about this to me yesterday when I had the same question

Reply:
Shouldn't they then instead use `$(NetCoreAppToolCurrent)` which is always in sync with the sdk? Either that or making them dependent on the live built targeting pack which I assume we don't want.

Reply:
Not sure.  That's a totally different framework version: what if they need to have coherent assembly references (I don't know if this is the case or not)?  IMHO these should be building against and testing the live product instead of an LKG but that might be bigger fish to fry.  @Anipik can you comment on the extent of the dependencies on `net6.0` in installer: can we dial those back to `net5.0` without changing what we ship?

Reply:
i am just trying that out

Reply:
to close on this, most components in installer don't need implicit framework references and hence using the latest TargetFramework works fine in these cases. `Microsoft.NET.HostModel` will be moved into the libraries section anyway. The remaining test projects presumably don't need to target a `$(NetCoreAppCurrent)` targeting pack and are fine with latest - 1.